frontend/fingertips-frontend/components/molecules/Export/export.types.ts

```diff
@@ -8,3 +8,19 @@ export interface ExportDownload {
   canvas?: HTMLCanvasElement;
   svg?: string;
 }
+
+export enum CsvHeader {
+  IndicatorId = 'Indicator Id',
+  IndicatorName = 'Indicator Name',
```

Comment:
```suggestion
  IndicatorName = 'Indicator name',
```
Just for consistency with the casing of the other columns/the spreadsheet.